contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -459,6 +464,9 @@ CREATE OR REPLACE PROCEDURE sys.sp_columns_100 (
     "@fusepattern" smallint = 1)
 AS $$
 BEGIN
+	-- TODO: we should be able to get rid of babelfish_truncate_identifier when we fix BABEL-5416
```

Comment:
I see the definition of procedure `sys.sp_columns_100` is copy of `sys.sp_columns`, should we update the definition of `sys.sp_columns` as well?

Reply:
Done